helm/thingsboard/templates/node.yaml

```diff
@@ -123,6 +123,8 @@ spec:
           {{- end }}
           - name: DEFAULT_INACTIVITY_TIMEOUT
             value: {{ .Values.node.state.defaultInactivityTimeoutInSec | default 600 | quote }}
+          - name: DEFAULT_CHECK_INTERVAL
+            value: { { .Values.node.state.defaultStateCheckIntervalInSec | default 60 | quote } }
```

Comment:
just curious: is the space in `{ {` needed compared to `{{` in the block above or is just ignored by Helm?

Reply:
Just a mistake I didn't notice, fixed!